pkg/preflight/preflight_utils.go

```diff
@@ -643,3 +668,46 @@ func deleteServiceAccount(clientset *kubernetes.Clientset, namespace string, ser
 	}
 	fmt.Printf("Deleted ServiceAccount: %s\n\n", serviceAccount.Name)
 }
+
+func createPreflightTestSecret(clientset *kubernetes.Clientset, namespace, secretName string, secretData []byte) (*apiv1.Secret, error) {
+	var secret *apiv1.Secret
+	var err error
+	// build the secret defination we want to create
+	secretSpec := &apiv1.Secret{
+		ObjectMeta: v1.ObjectMeta{
+			Name:      secretName,
+			Namespace: namespace,
+			Labels: map[string]string{
+				"app": "demo",
```

Comment:
is it required ??

Reply:
not necessary in this situation, but just retained it to follow the pattern I'm already following. Also helps with association, hence. Will change the value to "preflight" though.